JitsiConferenceEventManager.js

```diff
@@ -72,6 +72,8 @@ JitsiConferenceEventManager.prototype.setupChatRoomListeners = function() {
 
     chatRoom.addListener(XMPPEvents.AUDIO_MUTED_BY_FOCUS,
         value => {
+            Statistics.analytics.sendEvent('conference.remotelymuted');
```

Comment:
Looks like the other events will use camel case, so maybe it should be "remotelyMuted".

Reply:
Fixed!